src/components/SurveyComponents/EngineeringActivities.tsx

```diff
@@ -0,0 +1,85 @@
+import { FormSelect, Navigation, SurveyStep } from '@components'
+import { useEffect, useRef } from 'react'
+
+const activitiesQuestions = {
+  title: 'Activities',
+  codingPercentage: {
+    type: 'MCQ',
+    question: 'What percentage of your week is spent in coding/testing/implementing?',
+    choices: ['0-20%', '20-40%', '40-60%', '60-80%'],
+    isRequired: false,
+  },
+  designingPercentage: {
+    type: 'MCQ',
+    question: 'What percentage of your week is spent in designing?',
+    choices: ['0-20%', '20-40%', '40-60%', '60-80%'],
+    isRequired: false,
+  },
+  meetingsPercentage: {
+    type: 'MCQ',
+    question: 'What percentage of your week is spent in meetings?',
+    choices: ['0-20%', '20-40%', '40-60%', '60-80%'],
+    isRequired: false,
+  },
+  mentoringPercentage: {
+    type: 'MCQ',
+    question: 'What percentage of your week is spent in mentoring?',
+    choices: ['0-20%', '20-40%', '40-60%', '60-80%'],
+    isRequired: false,
+  },
+}
```

Comment:
You can define `percentageChoices` as a variable and you can reuse it across different questions, making your code cleaner and easier to maintain.

```javascript
const percentageChoices = ['0-20%', '20-40%', '40-60%', '60-80%'];

 codingPercentage: {
    type: 'MCQ',
    question: 'What percentage of your week is spent in coding?',
    choices: percentageChoices,
    isRequired: false,
  },
  designingPercentage: {
    type: 'MCQ',
    question: 'What percentage of your week is spent in designing?',
    choices: percentageChoices,
    isRequired: false,
  },
  meetingsPercentage: {
    type: 'MCQ',
    question: 'What percentage of your week is spent in meetings?',
    choices: percentageChoices,
    isRequired: false,
  },
  mentoringPercentage: {
    type: 'MCQ',
    question: 'What percentage of your week is spent in mentoring?',
    choices: percentageChoices,
    isRequired: false,
  }
  
  ```